### DISCUSSION TITLE
Higher powered card compatibility

### DISCUSSION THREAD

**Discussion by harryaskham** [osy/XG_Mobile_Station#8] [Hardware]
Would you expect that a card with a higher power consumption than the tested 4070 TI would work, given an enclosure with a sufficiently powerful PSU, or are there other factors that would cause issues? Say a desktop 4090 with a 500-600w supply. I am curious to test this and see what the bottlenecking cost is in practice.

**Comment by osy** [comment]
Yeah if you use an external PSU, you can go as high as you want.
